src/node_alive/autostarter.py

```diff
@@ -0,0 +1,119 @@
+# ROS
+import os
+import roslaunch
+import rospkg
+import rospy
+from std_msgs.msg import String
+
+# TU/e Robotics
+from node_alive.srv import AutoStarterCommand, AutoStarterCommandRequest, AutoStarterCommandResponse
+
+class AutoStarter(object):
+    """
+    Launches a launch file and waits for a command to either shutdown or to restart.
+    It is also possible to start other launch files that are stored on the parameter server, like the demo's.
```

Comment:
The docstring might not be entirely clear to people who're not involved in your project. What are "launch files that are stored on the parameter server" and what does "like the demo's" mean?

Reply:
rewritten it in a1ef39d19ac8416def3517506f5d571fcb9e1645